lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -152,6 +152,13 @@ def request_params
           params.delete('update_systems')
         end
 
+        add_content = {}
+        params['add_content'].each do |key, value|
+          add_content[key] = value if options.key?("option_#{key}")
```

Comment:
small correction: use `HammerCLI.option_accessor_name("foo")` you want to pull something from `options`.

so it would look like this:
```ruby
if options.key?(HammerCLI.option_accessor_name(key))
```